web/src/main/angular/src/app/core/components/transaction-id-search/transaction-id-search-container.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="l-search-group-wrap font-opensans">
```

Comment:
Remove unnecessary wrapping element.
```suggestion
```

Use other approaches to apply `font-opensans` class such as `HostBinding`